seaborn/categorical.py

```diff
@@ -1230,24 +1230,46 @@ def estimate_statistic(self, estimator, ci, n_boot):
             self.value_label = "{}({})".format(estimator.__name__,
                                                self.value_label)
 
-    def draw_confints(self, ax, at_group, confint, colors, **kws):
-
-        kws.setdefault("lw", mpl.rcParams["lines.linewidth"] * 1.8)
+    def draw_confints(self,
+                      ax,
+                      at_group,
+                      confint,
+                      colors,
+                      conf_lw=1,
+                      capsize=0,
+                      **kws):
+
+        kws.setdefault("lw", mpl.rcParams["lines.linewidth"] * conf_lw)
+        kws.pop('lw')
```

Comment:
I had to add this because when plotting with `pointplot`, another `lw` is explicitly specified, so I removed it from the keyword list. I guess with the way this is, there is no reason to have line 1242 at all?